public/js/Histogram.js

```diff
@@ -0,0 +1,100 @@
+var d3 = require('d3');
+var _ = require('lodash');
+_.mixin(require('lodash-deep'));
+
+function Histogram(sampleData, element, featureNames) {
```

Comment:
What does "prototypical inheritance" mean, other than being a cool name for a videogame? =P But, to reiterate, "ech". Point is, there's no real way to tell whether something is an "object" or a plain function until you start seeing "this" everywhere, right?